package.json

```diff
@@ -1,15 +1,19 @@
 {
   "private": true,
   "scripts": {
-    "build": "pulp build --to dist/app.js",
-    "watch": "pulp -w build --to dist/app.js",
-    "watch-fast": "webpack --mode=development --entry ./entry.js --output-path ./dist --output-filename app.js --progress --watch"
+    "postinstall": "spago install",
+    "test": "spago test",
+    "build": "spago build",
+    "bundle": "spago bundle-app --to dist/app.js",
+    "watch": "spago build --watch",
+    "bundle:watch": "spago bundle-app --watch --to dist/app.js",
```

Comment:
I'm not sure if there's a reason to provide this command - if you're developing with `purs ide` rather than having it watch while building, then there's probably no reason to have the slower bundle watch too?

I don't know, just thinking it might be less confusing to have one fewer option!

Reply:
I agree. I've trimmed things down a little.